hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -697,18 +693,6 @@ public void loadAll(Set<K> keys, boolean replaceExistingValues) {
         return dataKeys;
     }
 
-    @Override
-    public Set<K> keySet() {
-        MapKeySetRequest request = new MapKeySetRequest(name);
```

Comment:
@pveentjer Why did we changed keySet() , values() and entrySet() that does not use predicates ?

It seems they are replaced by QueryRequest using TruePredicate. I cant do the changes in c++ client since Predicates are not supported there.(because they are not Portable or IdentifiedDataSerializable) . I am not sure what is the situation with c# client.(There was a solution with full java class names, I wonder it works for this one)